### ISSUE TITLE
Debugger "failed to lookup source code" when function defined within function

### ISSUE THREAD

**Issue by zhopan77** [JuliaDebug/Debugger.jl#230]
In Julia REPL, define a function that contains another function's definition:

```
function main()
    todB(x) = 10*log10(x)
    println("100x is $(todB(100)) dB.")
end
```

Then in the REPL,
```
using Debugger
@enter main()
```
Debugger will not be able to use source code for debugging:
```
julia> @enter main()
failed to lookup source code, showing lowered code:
In main() at REPL[10]:2
 1  1 ─      todB = %new(var"#todB#4")
>2  │   %2 = (todB)(100)
 3  │   %3 = (string)("100x is ", %2, " dB.")
 4  │   %4 = (println)(%3)
 5  └──      return %4

About to run: (var"#todB#4"())(100)
1|debug>
```
If the function todB(x) is moved to global scope, Debugging has no problem to use source code.
This has been observed in both Julia 1.1.1 and Julia 1.3.0-rc3.0.

Here's Debugger's version information:
```
(v1.3) pkg> st Debugger
    Status `C:\Users\zpan\.julia\environments\v1.3\Project.toml`
  [a8cc5b0e] Crayons v4.0.0
  [31a5f54b] Debugger v0.6.2
  [aa1ae85d] JuliaInterpreter v0.7.4
```

Note the Juno IDE debugger (triggered by `Juno.@enter`) does not have this problem.

**Comment by KristofferC** [comment]
https://github.com/timholy/CodeTracking.jl/issues/50 is the core problem here.

**Comment by timholy** [comment]
Fixed by CodeTracking 0.5.10
